src/lib/types/stores.ts

```diff
@@ -60,4 +61,5 @@ export interface IUnleashStores {
     segmentStore: ISegmentStore;
     patStore: IPatStore;
     publicSignupTokenStore: IPublicSignupTokenStore;
+    suggetChangeStore: ISuggestChangeStore;
```

Comment:
This looks like a little typo 🦅 

```suggestion
    suggestChangeStore: ISuggestChangeStore;
```